src/SpendPermissionManager.sol

```diff
@@ -199,6 +208,20 @@ contract SpendPermissionManager is EIP712 {
         _;
     }
 
+    /// @notice Require that two spend permissions have the same `account` field.
+    ///
+    /// @param firstSpendPermission First spend permission to compare.
+    /// @param secondSpendPermission Second spend permission to compare.
+    modifier requireSameAccount(
```

Comment:
seems like this modifier is only going to be used once, so would prefer to have this check in the function itself for easier reading